### ISSUE TITLE
Question on Bal. in Table 2

### ISSUE THREAD

**Issue by e96031413** [dvirsamuel/DRAGON#2]
Hi, I find that you mention "Bal. refers to training DRAGON with a balanced loss."(last row in Table 2), which improves the performance significantly.

You also mentioned in section 6.5 "We further trained DRAGON with a balanced cross-entropy loss (Bal.)

I am wondering where does Bal. come from? Does it from a paper? If so, Could you please introduce me the paper?

**Comment by dvirsamuel** [comment]
Hi,
Balanced cross-entropy loss refers to a loss that encourages learning of tail classes using a non-uniform cost per misclassification. In other words, the loss would be larger for samples from the tail, compared to samples from the head.
It is a standard and traditional solution for unbalanced data. 

Our implementation of the loss is based on the scikit learn implementation which is based on [1]. See more [here](https://scikit-learn.org/stable/modules/generated/sklearn.utils.class_weight.compute_class_weight.html).

We compute the weights for each training sample [here](https://github.com/dvirsamuel/DRAGON/blob/629eab315d6f8c1c696ba13b488e334d5f8bc96f/attribute_expert/model.py#L393) and use it during training.

[1] Logistic Regression in Rare Events Data, King, Zen, 2001
